doc/decisions/gsettings_values.md

```diff
@@ -0,0 +1,52 @@
+# GSettings values
+
+## Issue
+
+`GSettings` uses `Glibs` `GVariant` type for it values. The `GVariant` type can hold many
+different variations of types but `Elektra` only distinguishes between strings
+and binary data.
+
+## Constraints
+
+ - saved values have to be readable by `GSettings` but also trough `Elektra` itself
+ - keep `Elektra` as `Glib` independent as possible
+ - values should be human readable
+
+## Assumptions
+
+ - `Elektras` strength is its modularity to provide multiple backends that decide how to save values
+ - `GVariants` can be printed as Strings and parsed back again if we know the type
+ - `GVariants` can be also serialized and deserialized
+ - representing `GVariants` values trough strings fulfills all three constraints
+
+## Considered Alternatives
+
+ - save serialized values as binarys,
+ - create a mapping between GVariants types and `spec` types and transform them when saving
+
+## Decision
+
+ - Use GVariants string representation
```

Comment:
Yes. You are right i probably should elaborate this problem better. I have to admit i totally forgot about Elektras c++ api and types while using the glib binding which are nearly identical as using the c api. Because of that I am probably also still lacking the proper knowledge about Elektras behavior with different types. This also brings up my question how the different plugin handle 'true' and 'false' strings. But I would rather like to discuss that at our next meeting so everyone benefits from it. I am already putting together a list inside #762 regarding other questions.

Again thank you for you thorough feedback, and sorry for my sometimes harsh answers.